.codecov.yml

```diff
@@ -0,0 +1,7 @@
+comment:
+  layout: "header, reach, diff, flags, files"
+  behavior: default
+  require_changes: false  # if true: only post the comment if coverage changes
+  require_base: no        # [yes :: must have a base report to post]
+  require_head: no       # [yes :: must have a head report to post]
+  branches: null
```

Comment:
invalid char at the ending of file

Reply:
This file is the same as that of pouch repo.

Reply:
well. I didn't see the `no newline at end of file` in pouch.